mjpc/tasks/humanoid/skateboard/steering.h

```diff
@@ -95,19 +95,14 @@ class Steering : public Task {
   void ModifyScene(const mjModel* model, const mjData* data,
                    mjvScene* scene) const override;
 
-  std::string Name() const override;
-  std::string XmlPath() const override;
-
  protected:
   std::unique_ptr<mjpc::ResidualFn> ResidualLocked() const override {
-    return std::make_unique<ResidualFn>(this, residual_.current_mode_,
-                                        residual_.reference_time_);
```

Comment:
This one really bit me. The class was not copied correctly which caused the id's set in the `ResetLocked` not get copied correctly.